test/runtime/kafka.go

```diff
@@ -110,24 +102,32 @@ var _ = Describe("RuntimeValidatedKafka", func() {
 		return err
 	}
 
-	BeforeEach(func() {
-		once.Do(initialize)
+	BeforeAll(func() {
+		logger = log.WithFields(logrus.Fields{"testName": "RuntimeKafka"})
```

Comment:
Test name does not match the actual test name. This should be made into a variable and used in both the `Describe` and this logger instantiation accordingly.